### ISSUE TITLE
What should we do with the data that can be fetched with the Manifest API ?

### ISSUE THREAD

**Issue by mssola** [SUSE/Portus#249]
As explained in the issue #200, we are trying to fetch data from the Manifest API. Some work has already been put to this in the PR #241, thanks to @lsamayoa . But there's some open questions: 
- What should we actually do with the fetched data ?
- Where should we make available this same data ?

You can take a look at the official Docker [Registry's API documentation](https://docs.docker.com/registry/spec/api/) to see what does the manifest offer to us.

This issue is open to anybody that has some ideas on this topic, so feel free to speak up :)

**Comment by holgerreif** [comment]
Triggered by the referenced #749 I have some thougts (even if this information does not come from the manifest). 
- Show the complete history (parents) of the image
- For each parent layer show information from the [image description](https://github.com/docker/docker/blob/master/image/spec/v1.md), esp. size, creation date and command (with the latter you can cycle trough the build steps), and provide a link ("previous") to the parent

**Comment by mssola** [comment]
@holgerreif I agree completely. I *really* want to see at least those two points implemented in the end result of the page.

**Comment by holgerreif** [comment]
* For each namespace the sum of images sizes should be displayed 
     (well actually this is a lie since shared base layers are counted twice)
   * To make it better: for each namespace collect all distinct used layers and sum up the sozes of each layer

**Comment by mssola** [comment]
@holgerreif sounds good to me.
